src/platforms/node/common/configuration/integrations/default-integrations.mdx

```diff
@@ -139,3 +139,36 @@ _(New in version 7.13.0.)_
 _Import name: `Sentry.Integrations.Modules`_
 
 This integration fetches names of all currently installed Node modules and attaches the list to the event. Once fetched, Sentry will cache the list for later reuse.
+
+### RequestData
+
+_(New in version 7.17.0.)_
+
+_Import name: `Sentry.Integrations.RequestData`_
+
+This integration adds data from the current incoming request (if any) to events.
```

Comment:
```suggestion
This integration adds data from an incoming request, (if one is in progress), to events.
```
Would like to simplify to just say "incoming request" unless it's commonly expected to call it "current incoming request" :)

Reply:
Okay, tried again. LMK what you think.